src/lib/style/theme.js

```diff
@@ -188,6 +182,14 @@ export const spacing = {
   sp28: '2rem',
   sp32: '2.286rem',
   sp40: '2.857rem',
+  sp47: '3.357rem',
+  sp60: '4.286rem',
+  sp65: '4.643rem',
+  sp70: '5rem',
+  sp75: '5.357rem',
+  sp100: '7.143rem',
+  sp120: '8.571rem',
+  sp150: '10.714rem'
```

Comment:
hello @hug33k ,

I suggest that for the ones we haven't defined on core-ui side, we hard code the space **in rem** on the UIs(MetalK8s/ xcore) side.

Because they are not standard spacing that we use in design, so i think it's ok to hardcode them. 

Thanks for addressing it ;)